src/App.jsx

```diff
@@ -1,12 +1,31 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import Home from "./containers/home";
 import Login from "./containers/login";
+import { useSelector } from "react-redux";
+import { auth } from "database";
+import Profile from "containers/profile";
 
 function App() {
-  return (
+  console.log();
+  const [user, setUser] = useState({});
+
+  useEffect(() => {
+    setUser(userFromState);
+  }, []);
+
+  const userFromState = useSelector((state) => state.users.user);
+  return !userFromState.full_name ? (
     <div className="flex justify-center items-center">
       <Login />
     </div>
+  ) : !userFromState.username ? (
+    <div className="flex justify-center items-center">
+      <Profile />
+    </div>
+  ) : (
+    <div className="flex justify-center items-center">
```

Comment:
If you have three conditions, it's better to use if statements, rather than a ternary operator. I would recommend keeping ternary operators for only 2 conditions. Makes it more readable with if statements

Reply:
done.